src/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="shortcut icon" href="favicon.ico" type="image/x-icon" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Personal portfolio</title>
+  </head>
+  <body>
+    <header class="container">
+      <div class="logo">
+        <h1>Bekjon</h1>
+      </div>
+      <div class="hamburger">
+        <div class="line"></div>
+        <div class="line"></div>
+        <div class="line"></div>
+      </div>
```

Comment:
- [ ] Good job implementing the links inside the header, but kindly wrap the hamburger with a `nav` element. Please note that in your project, the header should contain only the logo and the nav, and then nav links should contained inside the nav.